Guide to set up a Data Only Hotspot using a WisLink RAK LPWAN concentrator module and a Raspberry Pi

johansmacias opened the pull request:
This guide covers all the steps needed to set up a Data Only Hotspot using a WisLink RAK LPWAN concentrator module and a Raspberry Pi. This configuration can be achieved by installing the helium_gateway service available on the gateway-rs repository.

johansmacias committed:
Add rak-concentrators.mdx and images.

johansmacias committed:
Add the rak-concentrators.mdx to the sidebar.

johansmacias committed:
Update light-hotspots.mdx to add a link to the guide rak-concentrators.mdx.

jthiller commented on docs/use-the-network/light-hotspots/guides/rak-concentrators.mdx:
```diff
@@ -0,0 +1,377 @@
+
+:::warning
+Data Only Hotspots cannot earn $HNT, at least for now, and after milestone 2
+they will earn $HTN only for Device Packet Transfer. Remember to keep an eye on
```
```suggestion
they will earn $HNT only for Device Packet Transfer. Remember to keep an eye on
```

jthiller commented on docs/use-the-network/light-hotspots/guides/rak-concentrators.mdx:
```diff
@@ -0,0 +1,377 @@
+
+At this point, your Data Only Hotspot is ready to be added to the blockchain!
+You can find more information about that process
+[here](http://localhost:3000/mine-hnt/data-only-hotspots#pre-requisites).
```
```suggestion
[here](/mine-hnt/data-only-hotspots#pre-requisites).
```

johansmacias committed:
Apply suggestions from code review

It solves some typos on $HNT writing and a broken URL with localhost conflicts.

Co-authored-by: Joey Hiller <jthiller@gmail.com>

PharkMillups reviewed (dismissed):
Fantastic! Thanks. Merging. Ping me on discord so I can give you the docs contrib role.

PharkMillups committed:
Merge branch 'master' into master
